share/config_settings.toml

```diff
@@ -0,0 +1,145 @@
+version = "1.0"
+i18n = "global_settings_setting"
+
+[security]
+name = "Security"
+    [security.password]
+    name = "Passwords"
+        [security.password.admin_strength]
+        type = "number"
+        default = 1
+
+        [security.password.user_strength]
+        type = "number"
+        default = 1
+    
+    [security.ssh]
+    name = "SSH"
+        [security.ssh.ssh_compatibility]
+        type = "select"
+        default = "modern"
+        choices = ["intermediate", "modern"]
+
+        [security.ssh.ssh_port]
+        type = "number"
+        default = 22
+
+        [security.ssh.ssh_password_authentication]
+        type = "boolean"
+        yes = "True"
+        no = "False"
+        default = "False"
```

Comment:
I guess all those are related to the fact that in bash scripts we later check `if [[ $value == "True" ]]` (or similarly in jinja templates) ? I wonder what happens if we just set `default = true`, will `yunohost setting get foo.bar` output `true` ? I guess we could 'clean' the bash scripts and jinja template, but maybe that's not worth it ... (I'm just thinking out loud)